src/mattack_actors.cpp

```diff
@@ -304,7 +304,7 @@ bool gun_actor::call( monster &z ) const
     if( z.friendly ) {
         int max_range = 0;
         for( const auto &e : ranges ) {
-            max_range = std::max( { max_range, e.first.first, e.first.second } );
+            max_range = std::max( std::max( max_range, e.first.first ), e.first.second );
```

Comment:
I think it was `<algorithm>`. I assume it uses `accumulate` internally.
http://www.cplusplus.com/reference/algorithm/max/ says `max` requires `algorithm`, but they only use the 2 value one. We may have a forward declaration of the 2 argument max somewhere, possibly in the compiler itself.